### ISSUE TITLE
Install or Update to NuGet WebView2 0.9.538 - WebView2 not in Visual Studio ToolBox

### ISSUE THREAD

**Issue by mikemeinz** [MicrosoftEdge/WebViewFeedback#249]
I have tried two solutions (a total of four projects). Updating from WebView2 0.9.515-prerelease to 0.9.538. Also, tried uninstall and install of NuGet WebView2 0.9.538.. 

WebView2 doesn't show in Visual Studio ToolBox. 
All references to WebView2 methods, events and properties yield errors.

`using Microsoft.Web.WebView2.Core;`
causes this error
`Error	CS0234	The type or namespace name 'Web' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)	WebView2	C:\Users\MM\Documents\Visual Studio 2017\Projects\WebView2Test\WebView2\WebView.cs	
`

WebView2 0.9.538 NuGet package is missing the **lib** directory.
```
 Directory of C:\Users\MM\.nuget\packages\microsoft.web.webview2\0.9.538

06/08/2020  01:45 PM    <DIR>          .
06/08/2020  01:45 PM    <DIR>          ..
06/08/2020  01:45 PM               130 .nupkg.metadata
06/08/2020  10:32 AM            18,696 .signature.p7s
06/08/2020  01:45 PM    <DIR>          build
12/19/2019  01:31 AM             1,487 LICENSE.txt
06/08/2020  01:45 PM         1,152,802 microsoft.web.webview2.0.9.538.nupkg
06/08/2020  01:45 PM                88 microsoft.web.webview2.0.9.538.nupkg.sha512
06/05/2020  09:10 AM               943 microsoft.web.webview2.nuspec
06/08/2020  01:45 PM    <DIR>          package
06/05/2020  01:10 AM           129,588 WebView2.idl
06/05/2020  01:35 AM            40,824 WebView2.tlb
06/08/2020  01:45 PM    <DIR>          _rels
```

Visual Studio 2017 Version 15.9.22

**Comment by michael-russin** [comment]
Looks like they also published a **0.9.538-prerelease** that has the .net support.    That worked for me.

**Comment by pagoe-msft** [comment]
@mikemeinz, for now the .NET APIs will be in the pre-release package. We will post in the release notes when we move .NET APIs to the release package.
